azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/implementation/ApplicationStackInner.java

```diff
@@ -16,6 +16,87 @@
  * Application stack.
  */
 public class ApplicationStackInner {
+    /**
+     * Class representing the inner properties of an application stack.
```

Comment:
We don't modify generated classes. Reason for this change?

Reply:
The json data coming in from the service is not being accurately parsed... i had to add Properties for the data to be correctly set.